regression/rt_gk_neut_d3d_2x2v_p1.c

```diff
@@ -0,0 +1,1234 @@
+          .type_self = GKYL_SELF_ELC,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc",
+          .ion_nm = "ion",
+          .recvr_nm = "D0",
+    	  .charge_state = 0,
+          .ion_mass = ctx.mi,
+          .elc_mass = ctx.me,
+        },
+      },
+    },
+
+    .diffusion = {
+      .num_diff_dir = 1,
+      .diff_dirs = { 0 },
+      .D = { 0.1 },
+      .order = 2,
+    },
+    
+    .bcx = {
+      .lower={.type = GKYL_SPECIES_ABSORB,},
+      .upper={.type = GKYL_SPECIES_ABSORB,},
+    },
+    .bcy = {
+      .lower={.type = GKYL_SPECIES_GK_SHEATH,
+      },
+      .upper={.type = GKYL_SPECIES_GK_SHEATH,
+      },
+    },
+
+    .num_diag_moments = 1,
+    .diag_moments = { "BiMaxwellianMoments" },
+  };
+
+  // ions
+  struct gkyl_gyrokinetic_species ion = {
+    .name = "ion",
+    .charge = ctx.qi, .mass = ctx.mi,
+    .lower = { -1.0/sqrt(2.0), 0.0},
+    .upper = {  1.0/sqrt(2.0), 1.0},
+    .cells = { cells_v[0], cells_v[1] },
+    .polarization_density = ctx.n0,
+    
+    .mapc2p = {
+      .mapping = mapc2p_vel_ion,
+      .ctx = &ctx,
+    },
+
+    /* .init_from_file = { */
+    /*   .type = GKYL_IC_IMPORT_F, */
+    /*   .file_name = "gk_neut_d3d_2x2v_p1-ion_restart.gkyl", */
+    /* }, */
+
+    .correct = {
+      .correct_all_moms = true,
+      .use_last_converged = true,
+      .iter_eps = 1e-12,
+      .max_iter = 10,
+    },
+    
+    .projection = {
+      .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM,
+      .ctx_density = &ctx,
+      .ctx_upar = &ctx,
+      .ctx_temp = &ctx,
+      .density = density_init,
+      .upar = upar_ion,
+      .temp = temp_ion,
+    },
+
+    .collisions =  {
+      .collision_id = GKYL_LBO_COLLISIONS,
+      .ctx = &ctx,
+      .self_nu = nuIon,
+      .num_cross_collisions = 1,
+      .collide_with = { "elc" },
+    },
+
+    .source = {
+      .source_id = GKYL_PROJ_SOURCE,
+      .num_sources = 1,
+      .projection[0] = {
+        .proj_id = GKYL_PROJ_MAXWELLIAN_PRIM,
+        .ctx_density = &ctx,
+        .ctx_upar = &ctx,
+        .ctx_temp = &ctx,
+        .density = density_srcOMP,
+        .upar = zero_func,
+        .temp = temp_ion_srcOMP,
+      },
+    },
+
+    .react_neut = {
+      .num_react = 3,
+      .react_type = {
+        { .react_id = GKYL_REACT_IZ,
+          .type_self = GKYL_SELF_ION,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc",
+          .ion_nm = "ion",
+          .donor_nm = "D0",
+    	  .charge_state = 0,
+          .ion_mass = ctx.mi,
+          .elc_mass = ctx.me,
+        },
+    	{ .react_id = GKYL_REACT_RECOMB,
+          .type_self = GKYL_SELF_ION,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc",
+          .ion_nm = "ion",
+          .recvr_nm = "D0",
+    	  .charge_state = 0,
+          .ion_mass = ctx.mi,
+          .elc_mass = ctx.me,
+        },
+    	{ .react_id = GKYL_REACT_CX,
+          .type_self = GKYL_SELF_ION,
+          .ion_id = GKYL_ION_H,
+    	  .elc_nm = "elc", // gets called for other rxn. fix this?
```

Comment:
Why do we have to pass the electron mass to CX?

Reply:
Typo. Resolved in last commit.